### PR TITLE
set test location relative path relative to original test path

### PR DESCRIPTION
**Pull request by icemachined**
to be in sync with save-cloud logic

### CODE REVIEW

File: save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt [hunk 1]
```diff
@@ -56,7 +57,6 @@ class Save(
     fun performAnalysis(): Reporter {
         logInfo("Welcome to SAVE version $SAVE_VERSION")
         // FixMe: now we work only with the save.toml config and it's hierarchy, but we should work properly here with directories as well
```
Comment (petertrr) [inline save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt:59]: I think this comment is already outdated and can be removed. Logic with directories is partially implemented and will be improved under #409

**Review by petertrr — APPROVED** [review]
So with this change we'll have full paths in save-cloud execution history? I.e. `examples/kotlin-diktat/warn/chapter1` where we now have just `warn/chapter1`?

**Review by nulls — APPROVED** [review]

**Review by nulls — CHANGES_REQUESTED** [review]
please rebase with latest changes and check a new version of `SaveProperties`

File: save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt [hunk 2]
```diff
@@ -147,12 +151,10 @@ class Save(
         logDebug("=> Executing plugin: ${plugin::class.simpleName} for [${plugin.testConfig.location}]")
         reporter.onPluginExecutionStart(plugin)
         try {
-            val testRepositoryRootPath = plugin.testConfig.getRootConfig().location
-
             plugin.execute()
                 .onEach { event ->
                     // calculate relative paths, because reporters don't need paths higher than root dir
-                    val resourcesRelative = event.resources.withRelativePaths(testRepositoryRootPath)
```
Comment (nulls) [inline save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt:155]: previously we took it from plugin -- do you sure that it has the same value as from `saveProperties`?

Comment (icemachined) [reply save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt:155]: here we change logic to be compliant with the same logic from save-cloud, in save cloud it expect tests in folders which are  relative to testRootPath, when this path is not the same as test repository root path when we cant download debug info in gui

**Review by nulls — APPROVED** [review]

**Pull request merged**